src/gui/optionsdlg.ui

```diff
@@ -213,6 +213,16 @@
                 </widget>
                </item>
                <item>
+                <widget class="QCheckBox" name="checkLargeStatusIcons">
```

Comment:
Does this little tune really belong to the main options page? I would put it into the advanced page (where we have some icons-related settings already).